src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommandHandler.cs

```diff
@@ -19,22 +18,21 @@ public sealed class SubmitCommitmentCommandHandler : AsyncRequestHandler<SubmitC
         private readonly IEmployerCommitmentApi _commitmentApi;
         private readonly IMediator _mediator;
```

Comment:
I think we can get rid of _mediator now from here, which is good.

Reply:
sure, will fix